docs/ocs/troubleshooting.md

```diff
@@ -0,0 +1,169 @@
+# Troubleshooting OpenShift Container Storage (OCS)
+
```

Comment:
Can we also have a Pre-requisite section? In this case it would be cluster admin to the cluster hosting ocs?